### DISCUSSION TITLE
BetterDiscord Support

### DISCUSSION THREAD

**Discussion by AbyssalKaz** [maximmax42/Discord-CustomRP#165] [General]
Hey! Not really sure where to post this but I was wondering if you could try and get it working for BetterDiscord again, It worked perfectly fine roughly a week ago but now it's not working at all. Any kind of response would be nice! :D

As of current if you use BetterDiscord it gets stuck on "Connection Failed (Is Discord running?)"

**Comment by maximmax42** [comment]
It's not exactly in my hands I don't think. It might be some plugin that's screwing up the connection. Currently the only solution to this problem I found is to uninstall BD, connect with CRP at least once and then install BD back.
